code/modules/mailorder/modr_catalogues.dm

```diff
@@ -0,0 +1,323 @@
+//catalogue cartridges: the main event
+/obj/item/disk/data/cartridge/catalogue
+	name = "\improper unprogrammed mail-order cartridge"
```

Comment:
```suggestion
	name = "unprogrammed mail-order cartridge"
```
Explicit `\improper` only matters when the first letter is uppercase as that's a proper noun by default.